src/commands/Environment.ts

```diff
@@ -337,4 +342,15 @@ export
       });
     }
   }
+
+  envExists(): boolean {
+    if (!RxFs.exist(`${process.cwd()}/env/`)) {
+      return false;
+    }
+    return this.envCollection.collection.length < 0 ? false : true;
```

Comment:
- sollte hier keiner-gleich <= sein?
- das exists im file system braucht man eigentlich nicht, oder?
- return this.envCollection.collection.length > 0;